lib/generator/print_helpers.dart

```diff
@@ -210,13 +210,15 @@ Spec generateArgumentClassSpec(QueryDefinition definition) {
               ..type = refer('Map<String, dynamic>')
               ..name = 'json',
           ))
+          ..annotations.add(CodeExpression(Code('override')))
           ..body = Code('_\$${definition.className}ArgumentsFromJson(json)'),
       ))
       ..methods.add(Method(
         (m) => m
           ..name = 'toJson'
           ..lambda = true
           ..returns = refer('Map<String, dynamic>')
+          ..annotations.add(CodeExpression(Code('override')))
```

Comment:
info: these two methods override `JsonSerializable`'s implementation and were just missing the annotation